enigma/src/main/java/org/quiltmc/enigma/api/translation/mapping/tree/EntryTreeUtil.java

```diff
@@ -0,0 +1,33 @@
+package org.quiltmc.enigma.api.translation.mapping.tree;
+
+import org.quiltmc.enigma.api.translation.mapping.EntryMapping;
+
+public class EntryTreeUtil {
```

Comment:
don't love making a util class, but I don't want to stick this in `EntryTree` since it's locked to the `EntryMapping` generic